hop/auth.py

```diff
@@ -115,6 +142,10 @@ def _main(args):
         if os.path.exists(authfile) and not args.force:
             logger.warning("configuration already exists, overwrite file with --force")
         else:
+            logger.info("generating configuration with user-specified username + password")
             os.makedirs(os.path.dirname(authfile), exist_ok=True)
+
+            user = input("Username: ")
             with open(authfile, "w") as f:
-                f.write(DEFAULT_AUTH_CONFIG)
+                f.write(DEFAULT_AUTH_CONFIG.format(username=user, password=getpass.getpass()))
```

Comment:
Kinda feels like this should be a `toml.dump` call... I guess the advantage of this is that you get a `DEFAULT_AUTH_CONFIG` value which helps to document things?

Reply:
You're right, this is left over from the previous setup writing .ini files to disk. I can switch this to pass a dict to `toml.dump`.

Reply:
Fixed in the latest commit.